student-work/assignment-4/sofiafelan.md

```
@@ -0,0 +1,24 @@
+# Sofia Felan
+Hi, my name is Sofia Felan and my favorite programming language is Python because it is the first one I learned and easy to understand.
+
+## Example code
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)